services/cargohold/src/CargoHold/API/V3.hs

```diff
@@ -51,14 +53,14 @@ import qualified Data.Conduit.Binary     as Conduit
 import qualified Data.Text.Ascii         as Ascii
 import qualified Data.Text.Lazy          as LT
 
-upload :: V3.Principal -> Source IO ByteString -> Handler V3.Asset
+upload :: V3.Principal -> Source (ResourceT IO) ByteString -> Handler V3.Asset
```

Comment:
why is this needed?

Reply:
You mean the change from `IO` to `ResourceT IO`? That's the type required by `amazonka`

Reply:
:+1: